packages/GA4Client/test/src/tests.js

```diff
@@ -1,3 +1,5 @@
+var sinon = require('sinon');
```

Comment:
This works in a node environment, which is fine for now since `npm run test` tests things in node. However, it breaks testing in a chrome browser for easier debugging since there is no `require` keyword client side. The way to fix this is to create a build of the `tests.js` output, which is out of scope, so I recommend making a Jira ticket to support this in the future.  Alternatively, the move to jest could also resolve this to remove the dependency on an open browser testing.

Reply:
I think I have a good workaround for this as we can use sinon in the browser.